ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js

```diff
@@ -74,7 +80,12 @@ export default function SkipSRPBackup({ handleClose }) {
         justifyContent={JustifyContent.center}
         margin={4}
       >
-        <i className="fa fa-exclamation-triangle fa-2x skip-srp-backup-popover__icon" />
+        <Icon
+          name={ICON_NAMES.DANGER}
+          size={ICON_SIZES.XL}
+          class="skip-srp-backup-popover__icon"
```

Comment:
Nit: It should be `className`
```suggestion
          className="skip-srp-backup-popover__icon"
```